src/status_im/contexts/wallet/create_account/new_keypair/keypair_name/view.cljs

```diff
@@ -9,12 +10,20 @@
     [utils.re-frame :as rf]))
 
 (def keypair-name-max-length 15)
+(defn has-emojis? [s] (boolean (re-find utils.emojilib/emoji-regex s)))
+(defn has-special-characters?
+  [s]
+  (if (empty? s)
+    false
+    (not (re-find #"^[a-zA-Z0-9\-_ ]+$" s))))
 
-(defn view
-  []
+
+(defn view-internal
+  [{:keys [theme]}]
   (let [keypair-name (reagent/atom "")]
     (fn []
-      (let [customization-color (rf/sub [:profile/customization-color])]
+      (let [customization-color (rf/sub [:profile/customization-color])
+            [error? set-error?] (rn/use-state false)]
```

Comment:
I think we should be careful not to mix reagent and react here. On line 23 you have a reagent atom. So choose whichever suits best here

Reply:
@J-Son89 fixed it !